model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java

```diff
@@ -1936,8 +1936,13 @@ public Stream<GroupModel> getTopLevelGroupsStream(Integer first, Integer max) {
     }
 
     @Override
+    @Deprecated
     public Stream<GroupModel> searchForGroupByNameStream(String search, Integer first, Integer max) {
-        return session.groups().searchForGroupByNameStream(this, search, first, max);
+        return searchForGroupByNameStream(search, false, first, max);
+    }
+
+    public Stream<GroupModel> searchForGroupByNameStream(String search, Boolean exact, Integer first, Integer max) {
```

Comment:
Is it possible to remove this method and use `session.groups().searchForGroupByNameStream(....)` in all places where this is used?

Reply:
It was actually my intention to do just that after comments on the previous PR and it slipped through the cracks